packages/utils/src/run-prettier.ts

```diff
@@ -13,6 +12,13 @@ import logger from 'webpack-cli/lib/utils/logger';
 
 export function runPrettier(outputPath: string, source: string): void {
     let prettySource: string = source;
+    let prettier;
+    try {
+        // eslint-disable-next-line node/no-extraneous-require
+        prettier = require('prettier');
+    } catch (err) {
+        throw new Error(`You need to install 'prettier' to scaffold'.\n${err}`);
```

Comment:
No errors - no `prettier` no fixes

Reply:
Generated file would unreadable though, being one liner.
Should I add a warning at least for it?

Reply:
Added warning